tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_download_feeds.yaml

```diff
@@ -156,6 +156,17 @@
     download_timeout: 120
     update_treshold_weeks: 3
 
+- name: 'ALAS-3'
```

Comment:
Why did you choose the name `ALAS-3`?
Shouldn't it be `ALAS-2022` because it represents the version?

Reply:
Done in commit [e2f6f](https://github.com/wazuh/wazuh-qa/pull/2868/commits/e2f6f4043630ba10e607337b768e58f92b97fdef)